zowe/zlux-editor
# Updated webpack in order to copy missing icons into web folder

Things to test:
1. try editor with a C or JS file, look for fold icons and syntax coloring
2. try editor with/without APIML route, to ensure that the URLs are interpreted correctly. It looks correct to me.
Signed-off-by: 1000TurquoisePogs <sgrady@rocketsoftware.com>
